push_notifications/models.py

```diff
@@ -109,12 +109,17 @@ def send_message(self, message, **kwargs):
 
 
 class APNSDeviceManager(models.Manager):
+	# Here's where we could keep the credentials for a JWT so we don't have
+	# to recreated it for every request
+	#
+	# self.credentials = ...
+	# This would require some API to create some credentials maybe...
```

Comment:
Please remove this comment; that's most likely not going to happen.

Reply:
Sure. I guess if someone doesn't want to generate the JWT credentials on every single request, they use the PyAPNS API.